app/services/npq/ehco/targeted_delivery_funding_eligibility_updater.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module NPQ
+  module Ehco
+    class TargetedDeliveryFundingEligibilityUpdater
+      REOPEN_DATE = Time.zone.parse("6 June 2022 12:00")
+
+      class << self
+        delegate :run, to: :new
+      end
+
+      def run
+        logger = Logger.new($stdout)
+        logger.info "Updating EHCO NPQ Applications, this may take a couple of minutes..."
+
+        ehco_npq_course = NPQCourse.find_by(id: "66dff4af-a518-498f-9042-36a41f9e8aa7")
```

Comment:
This will return the [NPQEYL](https://github.com/DFE-Digital/early-careers-framework/blob/develop/db/seeds/initial_seed.rb#L55) course, not the [EHCO](https://github.com/DFE-Digital/early-careers-framework/blob/develop/db/seeds/initial_seed.rb#L54).

Also, it may worth finding the course by its `identifier` instead of its `id`?